fluentui_persona/src/main/java/com/microsoft/fluentui/tokenised/persona/Avatar.kt

```diff
@@ -0,0 +1,222 @@
+package com.microsoft.fluentui.tokenised.persona
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.DpOffset
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.FluentTheme.themeMode
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.Icon
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarImageNA
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarInfo
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarSize
+import com.microsoft.fluentui.theme.token.controlTokens.AvatarTokens
+
+val LocalAvatarTokens = compositionLocalOf { AvatarTokens() }
+val LocalAvatarInfo = compositionLocalOf { AvatarInfo() }
+
+@Composable
+fun Avatar(person: Person,
+           size: AvatarSize = AvatarSize.Medium,
+           imageNAStyle: AvatarImageNA = AvatarImageNA.Standard,
```

Comment:
Should imageNAStyle be part of Avatar token ?

Reply:
AvatarImageNA is part of AvatarToken. 
imageNAStyle is user configurable and the value is passed to AvatarInfo

Reply:
Confirm with Pulak about a scenario where it need to be vary as per use case within the app.